libs/repos/src/module/test/get-postreqs.test.ts

```diff
@@ -1,51 +1,40 @@
-import { setup, t, Repo, teardown } from '@modtree/test-env'
-import { unique } from '@modtree/utils'
-import { In } from 'typeorm'
-import { db } from '@modtree/typeorm-config'
+import { ModuleRepository } from '@modtree/repos'
+import { mocks } from '@modtree/test-env'
+import '@modtree/test-env/jest'
+import { Module } from '@modtree/types'
 
-beforeAll(() => setup(db, { restore: false }))
-afterAll(() => teardown(db))
+jest.mock('../../base')
+jest.mock('../../module')
 
-describe('single query', () => {
-  it('returns an array', async () => {
-    await Repo.Module.getPostReqs(['MA2001']).then((res) => {
-      expect(res).toBeInstanceOf(Array)
-      t.postReqsCodes = res
-    })
-  })
+const fakeData: Record<string, Partial<Module>> = {
+  ABC1000: { fulfillRequirements: ['shared1', 'shared2', 'ABC1000_only'] },
+  XYZ1000: { fulfillRequirements: ['shared1', 'shared2', 'XYZ1000_only'] },
+  DAB1000: { fulfillRequirements: [] },
+}
+const moduleRepo = new ModuleRepository(mocks.db, fakeData)
```

Comment:
This section in particular.

You can now create fake `Partial` data, and just send it into the repo